src/main/java/com/aws/greengrass/mqttbridge/auth/MQTTClientKeyStore.java

```diff
@@ -80,11 +82,14 @@ public void init() throws KeyStoreException, CertificateGenerationException {
             throw new KeyStoreException("Unable to load keystore", e);
         }
 
-        GetCertificateRequestOptions options = new GetCertificateRequestOptions();
-        options.setCertificateType(GetCertificateRequestOptions.CertificateType.CLIENT);
-        GetCertificateRequest certificateRequest =
-                new GetCertificateRequest(MQTTBridge.SERVICE_NAME, options, this::updateCert);
-        clientDevicesAuthServiceApi.subscribeToCertificateUpdates(certificateRequest);
+        clientDevicesAuthServiceApi.subscribeToCertificateUpdates(clientCertificateRequest);
```

Comment:
this could be OK, but it is a bit asymmetric. Since you're subscribing from the constructor, but unsubscribing in shutdown. What if it was restarted, would it break?

Reply:
ok, got it with the diff showing more context. That's fine then.